app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_split = command.split(" ")
+    with (
```

Comment:
You seem to have forgotten to check for this condition:
It must do nothing if the user tries to copy the file to file with the same.